ql/src/java/org/apache/hadoop/hive/ql/ddl/process/show/compactions/ShowCompactionsDesc.java

```diff
@@ -31,9 +31,11 @@
 public class ShowCompactionsDesc implements DDLDesc, Serializable {
   private static final long serialVersionUID = 1L;
 
+  // @formatter:off
   public static final String SCHEMA =
       "compactionid,dbname,tabname,partname,type,state,hostname,workerid,enqueuetime,starttime,duration,hadoopjobid,errormessage#" +
       "string:string:string:string:string:string:string:string:string:string:string:string:string";
+  // @formatter:on
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

I think this is probably an artifact of your IDE.

Reply:
Removed.